always display awx_status_total

kdelee opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->

<!---
If you are fixing an existing issue, please include "related #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->
Always report some number (even 0) for each job status. This way we don't have null data in monitoring data this makes writing alerts and dashboards easier
fixes https://github.com/ansible/awx/issues/12965

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
 - API

AlanCoding commented on awx/main/analytics/metrics.py:
```diff
@@ -169,8 +169,9 @@ def metrics():
     statuses = all_job_data.get('status', {})
-    for status, value in statuses.items():
-        STATUS.labels(status=status).set(value)
+    states = ('pending', 'waiting', 'running', 'successful', 'failed', 'error', 'canceled')
```
`set(dict(UnifiedJob.STATUS_CHOICES).keys()) - set(['new'])`?

kdelee replied:
sounds good will do

kdelee committed:
always display awx_status_total

this way we don't have null data in monitoring data
this makes writing alerts and dashboards easier

kdelee commented:
currently passing tests and this is output:
```
# HELP awx_status_total Status of Job launched
# TYPE awx_status_total gauge
awx_status_total{status="waiting"} 0.0
awx_status_total{status="successful"} 0.0
awx_status_total{status="failed"} 0.0
awx_status_total{status="error"} 0.0
awx_status_total{status="pending"} 0.0
awx_status_total{status="canceled"} 0.0
awx_status_total{status="running"} 0.0
```

kdelee commented:
![image](https://user-images.githubusercontent.com/29239279/192855628-1a143397-db41-4222-8950-9c7c3978818e.png)
